frontend/ui/renderimage.lua

```diff
@@ -67,13 +66,6 @@ end
 -- @treturn BlitBuffer
 function RenderImage:renderImageDataWithMupdf(data, size, width, height)
     if not Mupdf then Mupdf = require("ffi/mupdf") end
-    -- NOTE: Kobo's fb is BGR, not RGB. Handle the conversion in MuPDF if needed.
-    if Mupdf.bgr == nil then
-        Mupdf.bgr = false
-        if Device:hasBGRFrameBuffer() then
-            Mupdf.bgr = true
-        end
-    end
```

Comment:
OK, so this is now done in CanvasContext, which is related to document rendering.
No risk of it not being done yet if, say, you haven't opened any document yet (like if you start in File manager, display covers or do some Wikipedia lokup from there and display a page with some images in the textwidget)?